src/plugins/btrfs.c

```diff
@@ -128,6 +128,16 @@ gboolean bd_btrfs_check_deps () {
         ret = ret && status;
     }
 
+    status = bd_utils_have_kernel_module ("btrfs", &error);
+    if (!ret) {
+        if (error) {
+            g_warning("Failed to check for btrfs kernel module availability: %s" , error->message);
+            g_clear_error (&error);
+        } else
+            g_warning ("The 'btrfs' kernel module is not available.");
+    }
+    ret = ret && status;
+
```

Comment:
I'm afraid we also have to do this at the beginning of most of the *btrfs_* functions.